package.json

```diff
@@ -52,6 +52,7 @@
     "src"
   ],
   "dependencies": {
+    "loglevel": "1.6.1",
```

Comment:
I think these are meant to be alphabetical 😝

Reply:
You could throw a script at it, which sorts a JSON by keys in alphabetical order ;-)